src/docs/resources/faq.md

```diff
@@ -364,11 +364,19 @@ and there is approximately 450.4&nbsp;KB of (compressed) ICU data.
 These numbers were measured using [apkanalyzer](https://developer.android.com/studio/command-line/apkanalyzer),
 which is also built into [Android Studio](https://developer.android.com/studio/build/apk-analyzer).
 
+On iOS, a release IPA of the same app has a download size of 10.8&nbsp;MB on an
+iPhone X as reported by Apple's App Store Connect. The IPA is larger than the
+APK mainly because Apple encrypts binaries within the IPA making the compression
+less efficient (see "iOS App Store Specific Considerations" in
+[Apple's documentation](https://developer.apple.com/library/archive/qa/qa1795/_index.html).
```

Comment:
Perhaps link directly to https://developer.apple.com/library/archive/qa/qa1795/_index.html#//apple_ref/doc/uid/DTS40014195-CH1-APP_STORE_CONSIDERATIONS? Also, this is an unmaintained Q&A, not documentation. Maybe we should have something like this:

(see the [iOS App Store Specific Considerations](https://developer.apple.com/library/archive/qa/qa1795/_index.html#//apple_ref/doc/uid/DTS40014195-CH1-APP_STORE_CONSIDERATIONS) section of [QA1795](https://developer.apple.com/library/archive/qa/qa1795)).